client/src/hooks/AuthContext.tsx

```diff
@@ -115,7 +115,7 @@ const AuthContextProvider = ({
         navigate('/login');
       },
     });
-  }, []);
+  }, [authConfig?.test, navigate, refreshToken, setUserContext]);
```

Comment:
The auth context changes could be breaking, have you tested the refreshToken mutation? It's not always necessary to include all dependencies when they are stable but not known to eslint.

Reply:
How do I test the refreshToken mutation?

Reply:
@eburnette set low values for these in your .env file

```
SESSION_EXPIRY=1000 * 30 # 30 seconds
REFRESH_TOKEN_EXPIRY=1000 * 60 # 1 min
```

session will trigger a refresh at 30 seconds if you perform an action requiring session (chat, switch convo, etc). 

Also, if you leave the page idle for > 1 min after first logging in, you should observe a redirect to login.

Reply:
Tried that but I didn't see any refreshes or redirects.